Sources/StreamChatUI/Utils/Array+Extensions.swift

```diff
@@ -9,4 +9,9 @@ extension Array {
         guard index >= 0, index < count else { return nil }
         return self[index]
     }
+    var middle: Element? {
```

Comment:
@parth-simformsolutions  What happened with odd number of arrays?
Does that satisfy your condition?
Please double check